packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -41,6 +41,8 @@ import { Reaction } from './buttons/Reaction';
 import { useSelector } from '../hooks/useSelector';
 import { capabilitySelector } from '../../CallComposite/selectors/capabilitySelector';
 import { callStatusSelector } from '../../CallComposite/selectors/callStatusSelector';
+/* @conditional-compile-remove(DNS) */
+import { _isSafari } from '../../CallComposite/utils';
```

Comment:
Out of interest, does DNS have a isSupported function like video effects does? Ideally we would just call that so we catch other OS configurations that aren't suppported and so that the value could be updated through ECS

Reply:
This is added already, now that I tested it. It is returning true for safari even when it is not supported. let me check with Prathmesh
<img width="474" alt="Screenshot 2024-09-11 at 11 48 55 AM" src="https://github.com/user-attachments/assets/0afb1797-796f-47c7-a07a-b38514fc8076">